### PR TITLE
Update Controller UG with 2.5 attributes

### PR DESCRIPTION
**Pull request by ianf77**
Correcting procedures to match 2.5 UI

Controller User Guide UI updates chapters 24-33

https://issues.redhat.com/browse/AAP-23674

### CODE REVIEW

File: downstream/modules/platform/proc-controller-create-notification-template.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@ Use the following procedure to create a notification template.
 .Procedure
 
 . From the navigation panel, select {MenuAEAdminJobNotifications}.
```
Comment (EMcWhinn) [inline downstream/modules/platform/proc-controller-create-notification-template.adoc:9]: I can't remember if we spoke about this, its in the section above too. The UI shows Administration > Notifiers, not Job Notifications, so we probably need to change the attribute.

File: downstream/modules/platform/ref-controller-isolation-functionality.adoc [hunk 2]
```diff
@@ -16,8 +16,7 @@ This can be changed in the *Job Execution Path* field of the *Jobs settings* pag
 AWX_ISOLATION_BASE_PATH = "/opt/tmp"
 ----
 
-If there are any additional directories that should specifically be exposed from the host to the container that playbooks run in, you can
-specify those in the *Paths to Expose to Isolated Jobs* field of the *Jobs settings* page, or in the REST API at `/api/v2/settings/jobs`:
+If there are any additional directories that should specifically be exposed from the host to the container that playbooks run in, you can specify those in the *Paths to Expose to Isolated Jobs* field of the *Jobs settings* page, or in the REST API at `/api/v2/settings/jobs`:
```
Comment (EMcWhinn) [inline downstream/modules/platform/ref-controller-isolation-functionality.adoc:19]:
```suggestion
If there are any additional directories that should specifically be exposed from the host to the container that playbooks run in, you can specify those in the *Paths to expose to isolated jobs* field of the *Jobs Settings* page, or in the REST API at `/api/v2/settings/jobs`:
```

**Review by EMcWhinn — COMMENTED** [review]
A couple of very minor comments otherwise looks good.

File: downstream/modules/platform/proc-controller-create-notification-template.adoc (hunk 1 above)
Comment (ianf77) [reply downstream/modules/platform/proc-controller-create-notification-template.adoc:9]: I talked to Donna about this. I thought she'd fixed it.

Comment (ianf77) [reply downstream/modules/platform/proc-controller-create-notification-template.adoc:9]: I'll make the change.

**Review by EMcWhinn — APPROVED** [review]
LGTM

**Pull request merged**